pkg/kapp/clusterapply/change_set_view.go

```diff
@@ -48,3 +52,36 @@ func (v *ChangeSetView) Print(ui ui.UI) {
 func (v *ChangeSetView) Summary() string {
 	return v.changesView.Summary() // assumes Print was used before
 }
+
+func (v ChangeSetView) PrintCompleteYamlToBeApplied(ui ui.UI) error {
+
+	for _, view := range v.changeViews {
+		opSt := ""
+		if view.ApplyOp() == ClusterChangeApplyOpDelete {
+			st, _ := view.ApplyStrategyOp()
+			if st == deleteStrategyPlainAnnValue {
+				opSt = color.RedString("# %s: %s", view.ApplyOp(), view.Resource().Description())
+			} else {
+				opSt = color.RedString("# %s %s: %s", st, view.ApplyOp(), view.Resource().Description())
+			}
+			ui.PrintBlock([]byte(opSt + "\n"))
+			continue
+		}
+		opSt = color.GreenString("# %s: %s", view.ApplyOp(), view.Resource().Description())
+
+		resMgd := ctlres.NewResourceWithManagedFields(view.Resource(), false)
+		res, err := resMgd.Resource()
+		if err != nil {
+			return fmt.Errorf("Error: [%s]", err.Error())
+		}
+		by, err := res.AsYAMLBytes()
```

Comment:
hmm, should we be masking secrets?

Reply:
added masking resource in case of kind=secret